static/templates/rt_literal_property_attrs_doc.json

```diff
@@ -100,6 +100,50 @@
       }
     ]
   },
+  {
+    "@id": "_:b6",
+    "@type": ["http://sinopia.io/vocabulary/PropertyTemplate"],
+    "http://www.w3.org/2000/01/rdf-schema#label": [
+      {
+        "@value": "Literal property attributes"
+      }
+    ],
+    "http://sinopia.io/vocabulary/hasRemark": [
+      {
+        "@value": "Whether literal property is repeatable, required, ordered, etc."
```

Comment:
Needs to be changed.

Reply:
Sure, what text would you like?

Reply:
"Whether literal property has user ID default."